packages/sdk/src/pipeline.ts

```diff
@@ -0,0 +1,55 @@
+import { get, post, put, del, listen } from './request';
+import { readJson } from 'fs-extra';
+import { tunaMirrorURI } from './utils';
+
+export class Pipeline {
+  route: string;
+  constructor(host: string, port: number) {
+    this.route = `${host}:${port}/pipeline`;
+  }
+  async list(): Promise<any[]> {
+    return (await get(`${this.route}/list`)).rows;
+  }
+
+  async info(id: string): Promise<any> {
```

Comment:
Can we define the type of returned value?

Reply:
Done.